Readymade and Taidan guide for wiki (#106)
Ultramarine-Linux/wiki: src/content/docs/en/setup/installation.mdx

import Alert from "../../../../components/Docs/Alert.astro";

> If you are using an Anaconda image, proceed with the instructions below. If you are using a Readymade Preview image, go [here](#readymade).

Reviewer: ```suggestion
If you're using an Anaconda image, proceed with the instructions below. If you're using a Readymade Preview image, go [here](#readymade).
```
